### ISSUE TITLE
Idea: Half-size bins

### ISSUE THREAD

**Issue by rcarmo** [vector76/gridfinity_openscad#19]
I just saw this:

https://www.reddit.com/r/gridfinity/comments/xgn50x/i_have_expanded_my_half_bins_to_9_sizes/

...and it makes a weird kind of sense for some small, long parts (including electronics components, like headers).

Not a priority, but might be an interesting feature to add.

**Comment by vector76** [comment]
Huh, interesting.  So if I understand correctly, these
- Still fits on the normal 42 x 42 base cell size
- Only supports half cell width, not say 1.5 or 2.5 cell width
- Length is a whole multiple of cell size
- Stackable on itself
- Magnets and screws in the base probably don't make sense
- Looks like part of the lip is removed for better access

This is interesting.  I like this idea.  I will think about it.

**Comment by ostat** [comment]
I'm not sure I would see value in a half sized bin. 
Printing two half sized bins would use more plastic than one normal bin with a divider. 

That being said being able to set a size in half steps might be handy. Someone might need a size of 3.5x7.5 to fit a specific item, or fill out a draw.

Stacking would be an issue.

**Comment by vector76** [comment]
Ok I've made it flexible so it can accept fractional sizes for x (not for y, seems less important).
Added a drop-down to the customizer interface so 0.5 is easily accessible to customizer users without having to guess.
Other fractional values are allowed too, like 0.7 or 0.33 or 1.5.
This is added in commit 7eb04f7131b548b6ddc819d7f7ccfb5450ba0b41 so I'm closing this issue, but I'm still open to feedback, especially if I broke something.
